### DISCUSSION TITLE
Feedback for Introduction

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#90] [Page Feedback]
Add feedback for the Introduction page: https://csharp-develop.captaincoder.org/

**Comment by cloudthepain** [comment]
This looks good! I wonder if we should have a section in the intro that explains the different parts of the website?
  --- Lessons, this is where you learn
  --- Activities, this is where you get assignments
  --- Challenges, this is where you challenge yourself
  ---Journal, this ....

Just something to help them get an idea of the features of the website in a brief section.

**Reply by jcollard** [reply]
I agree. There is technically a section called *Book Features* which I would like to update and include all of these things. Or maybe even a section called *How to use this book*. Thoughts?

**Reply by cloudthepain** [reply]
Yeah, I think that would work pretty well!
